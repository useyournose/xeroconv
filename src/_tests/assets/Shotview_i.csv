"105 hybrid 31.0gt"
#,SPEED (FPS),Δ AVG (FPS),KE (FT-LB),POWER FACTOR (KGR⋅FT/S),TIME,CLEAN BORE,COLD BORE,SHOT NOTES
1, 2808.2, 13.9, , , 1:41:26 PM, , , "" 
2, 2790.7, -3.6, , , 1:41:52 PM, , , "" 
3, 2806.2, 12.0, , , 1:42:11 PM, , , "" 
4, 2800.3, 6.0, , , 1:42:33 PM, , , "" 
5, 2789.8, -4.5, , , 1:42:53 PM, , , "" 
6, 2790.1, -4.2, , , 1:43:09 PM, , , "" 
7, 2805.4, 11.1, , , 1:43:27 PM, , , "" 
8, 2778.0, -16.3, , , 1:43:41 PM, , , "" 
9, 2790.1, -4.2, , , 1:44:02 PM, , , "" 
10, 2789.1, -5.2, , , 1:44:18 PM, , , "" 
11, 2804.4, 10.1, , , 1:44:32 PM, , , "" 
12, 2795.1, 0.8, , , 1:47:08 PM, , , "" 
13, 2776.5, -17.8, , , 1:47:22 PM, , , "" 
14, 2797.0, 2.7, , , 1:48:06 PM, , , "" 
15, 2795.7, 1.4, , , 1:48:23 PM, , , "" 
16, 2786.9, -7.4, , , 1:48:43 PM, , , "" 
17, 2797.8, 3.5, , , 1:48:59 PM, , , "" 
18, 2793.0, -1.2, , , 1:49:17 PM, , , "" 
19, 2797.3, 3.0, , , 1:49:38 PM, , , "" 
-,,,,,,
AVERAGE SPEED,2794.3,,,,,,,
STD DEV,8.5,,,,,,,
SPREAD,31.7,,,,,,,
SESSION NOTE,"",,,,,,,
-,,,,,,
DATE, "April 6, 2024 at 1:36 PM",,,,,
All shots included in the calculations,,,,,,,﻿