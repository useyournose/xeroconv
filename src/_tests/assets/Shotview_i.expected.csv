sep=;
Device ID;useyournose-xeroconv;;

Series No;0001;;
Total number of shots;0019;;

Units velocity;fps;;
Units distances;yrds;;
Units kinetic energy;ft-lbs;;
Units weight;grains (grs);;

Stats - Average;2794.295;fps;
Stats - Highest;2808.2;fps;
Stats - Lowest;2776.5;fps;
Stats - Ext. Spread;31.7;fps;
Stats - Std. Dev;8.5;fps;

Shot ID;V0;Ke0;Proj. Weight;Date;Time
0001;2808.2;0;0;06-04-2024;1:41:26 PM;
0002;2790.7;0;0;06-04-2024;1:41:52 PM;
0003;2806.2;0;0;06-04-2024;1:42:11 PM;
0004;2800.3;0;0;06-04-2024;1:42:33 PM;
0005;2789.8;0;0;06-04-2024;1:42:53 PM;
0006;2790.1;0;0;06-04-2024;1:43:09 PM;
0007;2805.4;0;0;06-04-2024;1:43:27 PM;
0008;2778;0;0;06-04-2024;1:43:41 PM;
0009;2790.1;0;0;06-04-2024;1:44:02 PM;
0010;2789.1;0;0;06-04-2024;1:44:18 PM;
0011;2804.4;0;0;06-04-2024;1:44:32 PM;
0012;2795.1;0;0;06-04-2024;1:47:08 PM;
0013;2776.5;0;0;06-04-2024;1:47:22 PM;
0014;2797;0;0;06-04-2024;1:48:06 PM;
0015;2795.7;0;0;06-04-2024;1:48:23 PM;
0016;2786.9;0;0;06-04-2024;1:48:43 PM;
0017;2797.8;0;0;06-04-2024;1:48:59 PM;
0018;2793;0;0;06-04-2024;1:49:17 PM;
0019;2797.3;0;0;06-04-2024;1:49:38 PM;
